Services/Administration/OJS.Services.Administration.Data/IRoleDataService.cs

```diff
@@ -0,0 +1,8 @@
+﻿namespace OJS.Services.Administration.Data;
+
+using OJS.Data.Models.Users;
+using OJS.Services.Common.Data;
+
+public interface IRoleDataService : IDataService<Role>
```

Comment:
The convention is to name all the services in plural form - Roles..Service

Reply:
Fixed